src/View/Cell.php

```diff
@@ -183,9 +214,7 @@ public function render($template = null)
         };
 
         if ($cache) {
-            return $this->View->cache(function () use ($render) {
-                echo $render();
-            }, $cache);
+            return Cache::remember($cache['key'], $render, $cache['config']);
```

Comment:
The only way View::cache() can be used is if a new View instance is created inside the closure after action is invoked. Is that acceptable?